ext/nnstreamer/tensor_filter/tensor_filter_armnn.cc

```diff
@@ -252,6 +276,19 @@ ArmNNCore::makeTfLiteNetwork ()
   }
   return 0;
 }
+#else /* ENABLE_ARMNN_TFLITE */
+/**
+ * @brief A dummy function used when tensorflow-lite is not enabled at build-time.
+ * @param[in] input_map input data map
+ * @param[in] output_vec output data vector
```

Comment:
The written doxygen params seems to be wrong (also in lines 249-250).

Reply:
Fixed by inserting a comment-fix commit.